docs/tokenFormats/build.md

```diff
@@ -0,0 +1,497 @@
+# Build
+
+Token formats and restrictions for the `Build` type.
+
+## POST /build/{id}/schedule
+
+A supplier schedules a build of 1-10 parts, each of which is optionally tied to an order.
+
+| Inputs                               | Outputs                                     |
+| :----------------------------------- | :------------------------------------------ |
+| Recipe0... RecipeN, Order0... OrderN | Build, Recipe0... RecipeN, Order0... OrderN |
+
+### Request body
+
+#### Inputs
+
+`[RECIPE]`
+The latest token of each recipe in the build is consumed.
+
+`[ORDER]`
+The latest token of each order in the build is consumed. Optional.
+
+#### Outputs
+
+`[RECIPE]`
+So that each recipe is available to be used again, a new token for each recipe in the build is created.
+
+| Roles                | Metadata                   |
+| :------------------- | :------------------------- |
+| Buyer: `BAE`         | `<Literal>` type: `RECIPE` |
+| Supplier:`SupplierX` |                            |
+
+`[ORDER]`
+So that each order is available to be used again, a new token for each order in the build is created.
+
+| Roles                | Metadata                  |
+| :------------------- | :------------------------ |
+| Buyer: `BAE`         | `<Literal>` type: `ORDER` |
+| Supplier:`SupplierX` |                           |
+
+The `Build` token. Each `recipeN: <TokenId>` matches a `<TokenId>` from the `inputs`. Each `orderN: <TokenId>` matches a `<TokenId>` from the `inputs`.
+
+| Roles                | Metadata                                                          |
+| :------------------- | :---------------------------------------------------------------- |
+| Owner: `SupplierX`   | `<Literal>` type: `BUILD`                                         |
+| Buyer: `BAE`         | `<Literal>` status: `scheduled`                                   |
+| Supplier:`SupplierX` | `<Literal>` transactionId: `09000000-0000-1000-8000-000000000000` |
+|                      | `<Literal>` completionEstimate: `2023-01-01`                      |
+|                      | `<Literal>` externalId: `34-396589-2`                             |
+|                      | `<TokenId>` recipe0: `100`                                        |
```

Comment:
will need to increase `MaxMetadataCount` in the runtime if we want a token with these many items https://github.com/digicatapult/dscp-node/blob/61d0cbc78c7a40cb21449b1837792063895ae25e/runtime/src/lib.rs#L312